commands/scanpullrequest_test.go

```diff
@@ -395,6 +396,10 @@ func TestRunInstallIfNeeded(t *testing.T) {
 	scanSetup.SetFailOnInstallationErrors(true)
 	assert.NoError(t, runInstallIfNeeded(&scanSetup, ""))
 	tmpDir, err := fileutils.CreateTempDir()
+	defer func() {
+		err = fileutils.RemoveTempDir(tmpDir)
+		assert.NoError(t, err)
+	}()
 	assert.NoError(t, err)
```

Comment:
This error assertion should be placed before the defer clause, meaning right after the function that returns the error.